submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/service/FilePersistenceService.kt

```diff
@@ -50,9 +49,9 @@ class FilePersistenceService(
         val tsv = serializationService.serializeElement(element, SubFormat.TSV)
         val submissionPath = folderResolver.getSubmissionFolder(relPath)
 
-        FileUtils.writeStringToFile(submissionPath.resolve("$outputFileName.json").toFile(), json, Charsets.UTF_8)
-        FileUtils.writeStringToFile(submissionPath.resolve("$outputFileName.xml").toFile(), xml, Charsets.UTF_8)
-        FileUtils.writeStringToFile(submissionPath.resolve("$outputFileName.pagetab.tsv").toFile(), tsv, Charsets.UTF_8)
+        NfsFileUtils.copyOrReplace(submissionPath.resolve("$outputFileName.json").toFile(), json)
```

Comment:
I know this is something I should have pointed out in the previous PR but probably we don't want to call this `NfsFileUtils` since we'll be moving to ObjectStore (or who knows what in the future) so maybe it's better to name it `FileUtils`. Anyway, this is just me thinking out loud, I'll leave it at your discretion.

Reply:
yes agree, I did not use as this name is used by every single library so auto-complete guess confuse. 

I will give a try again